packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -733,7 +732,7 @@ export class GarbageCollector implements IGarbageCollector {
 				{
 					eventName,
 					category: "generic",
-					url: trimLeadingSlashes(toNodePath),
```

Comment:
Save this for another PR maybe?

Reply:
I did this so that I don't have to export trimLeadingSlases. Its only used in GCDataBuilder other than this.